pajbot/dispatch.py

```diff
@@ -412,10 +413,17 @@ def remindme(bot, source, message, event, args):
 
         delay = int(parts[0])
         reminder_text = " ".join(parts[1:]).strip()
-        extra_message = f"{source}, your reminder from {delay} seconds ago is over: {reminder_text}"
 
-        bot.say(f"{source}, I will remind you of '{reminder_text}' in {delay} seconds. SeemsGood")
-        bot.execute_delayed(delay, bot.say, extra_message)
+        if delay <= 30:
+            bot.say(f"{source}, You cannot set a reminder for less than 30 seconds!")
```

Comment:
says "less than" but the check is actually for "equal or less than". Setting reminder for 30 seconds wont work like this and the error would be confusing